src/duqtools/submit.py

```diff
@@ -143,8 +147,13 @@ def get_resubmit_jobs(resubmit_names: Sequence[Path]) -> List[Job]:
     return jobs
 
 
-def submit(*, force: bool, max_jobs: int, schedule: bool, array: bool,
-           resubmit: Sequence[Path], **kwargs):
+def submit(*,
+           force: bool,
+           max_jobs: int,
+           schedule: bool,
+           array: bool,
+           resubmit: Optional[Sequence[Path]] = None,
```

Comment:
resubmit (even if not specified with with `submit --resubmit` will be an empty tuple via the `duqtools cli`, so I would suggest to make that the default. `if resubmit` will still evaluate to `False` with an empty tuple, apparently.
```suggestion
           resubmit: Sequence[Path] = (),
```

Reply:
Yeah, I I wanted to signal that this is really optional, and that default is no resubmission, so I made it very explicit.